rabbitmq.info.yml

```diff
@@ -2,5 +2,4 @@ name: "RabbitMQ"
 description: "RabbitMQ Integration for Drupal"
 core: "8.x"
 type: "module"
-php: "5.6"
 package: "Performance and scalability"
```

Comment:
ISTR there is something which depends upon 5.6 in this version, maybe a structured constant initialization ? I can't find it again offhand : did you check if the module worked on an older version ? Without this line, requirement level is only 5.5.9 as per https://www.drupal.org/requirements/php#8